app/api/security/breeze_buddy/authorization.py

```
@@ -209,128 +213,127 @@ def has_wildcard_merchant_access(current_user: UserInfo) -> bool:
+            if requested_reseller_id not in accessible_resellers:
                 logger.warning(
-                    f"User {current_user.username} attempted to access unauthorized merchant: {requested_merchant_id}"
+                    f"User {current_user.username} attempted to access unauthorized reseller: {requested_reseller_id}"
                 )
                 raise HTTPException(
                     status_code=status.HTTP_403_FORBIDDEN,
-                    detail=f"Access denied to merchant {requested_merchant_id}",
+                    detail=f"Access denied to reseller {requested_reseller_id}",
                 )
-            merchant_filter = [requested_merchant_id]
+            [requested_reseller_id]
```

Ruff (0.14.14)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)